<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="src.RESTAPITests.Member.Creation.createMemberAcceptanceTest" tests="6" skipped="0" failures="5" errors="0" timestamp="2017-11-28T12:11:58" hostname="Bills-iMac.local" time="0.102">
  <properties/>
  <testcase name="createMultipleMembers" classname="src.RESTAPITests.Member.Creation.createMemberAcceptanceTest" time="0.001">
    <failure message="java.lang.NullPointerException" type="java.lang.NullPointerException">java.lang.NullPointerException
	at SETLAPIHelpers.RestAPI.MemberHelper.createMember(MemberHelper.java:86)
	at src.RESTAPITests.Member.Creation.createMemberAcceptanceTest.createMultipleMembers(createMemberAcceptanceTest.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:745)
</failure>
  </testcase>
  <testcase name="failToCreateMemberWithNullUserName" classname="src.RESTAPITests.Member.Creation.createMemberAcceptanceTest" time="0.0">
    <failure message="java.lang.NullPointerException" type="java.lang.NullPointerException">java.lang.NullPointerException
	at SETLAPIHelpers.RestAPI.MemberHelper.createMemberFailure(MemberHelper.java:69)
	at src.RESTAPITests.Member.Creation.createMemberAcceptanceTest.failToCreateMemberWithNullUserName(createMemberAcceptanceTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:745)
</failure>
  </testcase>
  <testcase name="failToCreateMemberWithEmptyStringUserName" classname="src.RESTAPITests.Member.Creation.createMemberAcceptanceTest" time="0.001">
    <failure message="java.lang.NullPointerException" type="java.lang.NullPointerException">java.lang.NullPointerException
	at SETLAPIHelpers.RestAPI.MemberHelper.createMemberFailure(MemberHelper.java:69)
	at src.RESTAPITests.Member.Creation.createMemberAcceptanceTest.failToCreateMemberWithEmptyStringUserName(createMemberAcceptanceTest.java:94)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:745)
</failure>
  </testcase>
  <testcase name="failToCreateMemberWithIncorrectPermissions" classname="src.RESTAPITests.Member.Creation.createMemberAcceptanceTest" time="0.0">
    <failure message="java.lang.NullPointerException" type="java.lang.NullPointerException">java.lang.NullPointerException
	at SETLAPIHelpers.RestAPI.MemberHelper.createMemberFailure(MemberHelper.java:69)
	at src.RESTAPITests.Member.Creation.createMemberAcceptanceTest.failToCreateMemberWithIncorrectPermissions(createMemberAcceptanceTest.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:745)
</failure>
  </testcase>
  <testcase name="failToCreateDuplicateUser" classname="src.RESTAPITests.Member.Creation.createMemberAcceptanceTest" time="0.003">
    <failure message="java.lang.NullPointerException" type="java.lang.NullPointerException">java.lang.NullPointerException
	at SETLAPIHelpers.RestAPI.MemberHelper.createMemberSuccess(MemberHelper.java:58)
	at src.RESTAPITests.Member.Creation.createMemberAcceptanceTest.failToCreateDuplicateUser(createMemberAcceptanceTest.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:745)
</failure>
  </testcase>
  <testcase name="createNewMember" classname="src.RESTAPITests.Member.Creation.createMemberAcceptanceTest" time="0.096"/>
  <system-out><![CDATA[semnd eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJtZXNzYWdlYm9keSI6eyJ1c2VyaWQiOjR9LCJtZXNzYWdldHlwZSI6ImluaXRpYWxpc2UiLCJyZXF1ZXN0aWQiOjEsInRva2VuIjoiIiwidGltZXN0YW1wIjoxNTExODcxMTE4NjUyfQ.XLotcDEV1OocLLPf1hnsWDNttYDqwZms-FLYaFWJkHksemnd eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJtZXNzYWdlYm9keSI6eyJ0b3BpYyI6Im5tIiwibWVtYmVyTmFtZSI6IlRlc3RfTWVtYmVyX0RvVFZJIiwiZW1haWwiOiJUZXN0X01lbWJlcl9Eb1RWSUB0ZXN0LmNvbSJ9LCJtZXNzYWdldHlwZSI6InJlcXVlc3QiLCJyZXF1ZXN0aWQiOjEsInRva2VuIjoiMDc4ZjZlOTczMjJlYThlYTlkOTdkNDliYmRiNzhjMzliMDVkODBjZjllY2M2NWM5MDg4ZGY0MzcyMjVhODJhZCIsInRpbWVzdGFtcCI6MTUxMTg3MTExODY2MH0.DMZFpDNeCUisi254l5GNXwGlXe-LminXhRVxaZVvs1E{Status=OK, memberID=233, userID=443, userName=Admin_233, emailAddress=Test_Member_DoTVI@test.com, accountID=259, parent=233, isManager=0, accountLocked=0, userType=35, isAdmin=0, accountName=Test_Member_DoTVI_Admin_Account, description=Test_Member_DoTVI admin account, billingWallet=, memberName=Test_Member_DoTVI, pass=JTGECB7Y5JY4}
]]></system-out>
  <system-err><![CDATA[eyJhbGciOiJIUzI1NiJ9.eyJyZXF1ZXN0aWQiOjEsIm1lc3NhZ2V0eXBlIjoiZGIiLCJzdGF0dXMiOiJPSyIsImRhdGEiOlt7IlN0YXR1cyI6Ik9LIiwibWVtYmVySUQiOjIzMywidXNlcklEIjo0NDMsInVzZXJOYW1lIjoiQWRtaW5fMjMzIiwiZW1haWxBZGRyZXNzIjoiVGVzdF9NZW1iZXJfRG9UVklAdGVzdC5jb20iLCJhY2NvdW50SUQiOjI1OSwicGFyZW50IjoyMzMsImlzTWFuYWdlciI6MCwiYWNjb3VudExvY2tlZCI6MCwidXNlclR5cGUiOjM1LCJpc0FkbWluIjowLCJhY2NvdW50TmFtZSI6IlRlc3RfTWVtYmVyX0RvVFZJX0FkbWluX0FjY291bnQiLCJkZXNjcmlwdGlvbiI6IlRlc3RfTWVtYmVyX0RvVFZJIGFkbWluIGFjY291bnQiLCJiaWxsaW5nV2FsbGV0IjoiIiwibWVtYmVyTmFtZSI6IlRlc3RfTWVtYmVyX0RvVFZJIiwicGFzcyI6IkpUR0VDQjdZNUpZNCJ9XX0.ulawThtuWQmlF9crIBaqKHJVCdAvj1_THTVRP6XeBKs
]]></system-err>
</testsuite>
